tensorflow_serving/g3doc/setup.md

```diff
@@ -46,14 +46,12 @@ apt-get remove tensorflow-model-server
 
 2.  Install and update TensorFlow ModelServer
 
-<!-- common_typos_disable -->
 
     ```shell
+    apt-get update
```

Comment:
I think making this one line (if it fits in 80 cols) would be nicer.
apt-get update && apt-get install...

Reply:
Done